client/my-sites/earn/ads/form-settings.jsx

```diff
@@ -159,94 +183,67 @@ class AdsFormSettings extends Component {
 					<FormRadio
 						name="show_to_logged_in"
 						value="pause"
-						checked={ 'pause' === this.state.show_to_logged_in }
-						onChange={ this.handleChange }
-						disabled={ this.props.isLoading }
+						checked={ 'pause' === settings.show_to_logged_in }
+						onChange={ handleChange }
+						disabled={ isLoading }
 						label={ translate( 'Pause ads (no revenue)' ) }
 					/>
 				</FormLabel>
 			</FormFieldset>
 		);
 	}
 
```

Comment:
The additionalAdsOption() method was not used in this component so I've deleted it.